app/views/home/index.html.erb

```diff
@@ -37,7 +41,63 @@
       <a href="#" class="button tiny">Search</a>
     </li>
 </li>
+</ul>
+</div>
 
 
-</ul>
+<div class="home-pics">
```

Comment:
I think it would be really cool to iterate through the locations that we create in our database and great these divs